Formula/felinks.rb

```diff
@@ -15,19 +14,20 @@ class Felinks < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c130f71e181e3a807ad0e1951b2b0f97301f7107829d21fb3817d263f918c977"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "groff" => :build
-  depends_on "libtool" => :build
-  depends_on "xmlto" => :build
```

Comment:
- `groff` -- I think this is for reading manpages, so isn't used at build. For Linux, would be included with `man-db`.
- `libtool` -- I didn't see any `(g)libtoolize` or any other reference to `libtool` so may be safe to drop.
- `xmlto` -- I think manpages are either already generated (`*.5`) or in a template format (`*.1.in`) and doesn't need the `asciidoc --> xmlto --> ...` build.